airbyte-webapp/src/components/ui/Card/Card.tsx

```diff
@@ -40,9 +42,12 @@ export const Card: React.FC<React.PropsWithChildren<CardProps>> = ({
             [styles.lightPadding]: lightPadding || !children,
             [styles.roundedBottom]: roundedBottom,
             [styles.withDescription]: description,
+            [styles.noLine]: noLine,
```

Comment:
What's the reason for having a card variant with and without a line under the title? Can we push back to have more consistency here, instead of adding two variants that are very similar?

Reply:
I like them so much more without the line, but reverted that and will take the convo to a channel if I don't hear back Monday. :)